src/formatDate.js

```diff
@@ -50,7 +50,60 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.find((item) => item.length === 1);
```

Comment:
```suggestion
  const oldSeparator = fromFormat[3];
```

- Overcomplicated
- If separator = ', ' (its length not 1) this way will not work

Reply:
actually, when separator on different index it's doesn't work too :)

Reply:
What if the separator will be `--`? Better use the index cause the array length is strictly defined by the task and you can rely on it.

Reply:
Yes, but as for me separator can be on index[1] or anywhere. What we need to do on this situation? And for this issues i found element with length = 1. Or i didn't right?

Reply:
You have task, task determines how data is prepared. Also, your solution doesn't work for separator longer than 1 char.